pkg/worker/worker.go

```diff
@@ -107,12 +107,14 @@ func (w *worker) Run() {
 	w.cancelToken <- true
 }
 
-func (w *worker) publishError(message pipeline.Message, err error) error {
-	errJSON, err := json.Marshal(workerError{
-		Topic:     w.mqQueue,
-		MessageID: message.ID,
-		Worker:    w.id,
-		Error:     err.Error(),
+func (w *worker) publishError(message pipeline.Message, attempt pipeline.Message, err error) error {
+	errJSON, err := json.Marshal(pipeline.PipelineError{
+		ReceivedByWorker:  message,
+		ProcessingAttempt: attempt,
+		Timestamp:         message.Timestamp,
+		Topic:             w.mqQueue,
```

Comment:
Topic =/= queue
Topic is the routing key which can be fetched  from the amqp091 delivery